src/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,18 @@
+import { LinkProps } from "@/constants/index.types";
+import Card from "./Card/Card";
+import styles from "./CardList.module.scss";
+
+interface CardListProps {
+  links?: LinkProps[] | undefined;
```

Comment:
옵셔널 프라퍼티(?기호) 자체가 undefined를 표현해주는 것이기 때문에 요렇게 해줘도 될 것 같네요~
```suggestion
  links?: LinkProps[];
```